recipes/libxml2/all/conanfile.py

```diff
@@ -132,19 +133,25 @@ def _build_mingw(self):
             configure_command = ' '.join(args)
             self.output.info(configure_command)
             self.run(configure_command)
+
+            self.run("mingw32-make -f Makefile.mingw")
+
+    def _package_mingw(self):
+        tools.mkdir(os.path.join(self.package_folder, "include", "libxml2"))
+        with tools.chdir(os.path.join(self._source_subfolder, 'win32')):
             self.run("mingw32-make -f Makefile.mingw install")
 
-    def _build_with_configure(self):
-        in_win = self.settings.os == "Windows"
-        env_build = AutoToolsBuildEnvironment(self, win_bash=in_win)
-        if not in_win:
-            env_build.fpic = self.options.fPIC
-        full_install_subfolder = tools.unix_path(self.package_folder) if in_win else self.package_folder
+    @lru_cache(1)
```

Comment:
After some discussion, we would like to remove this decorator as it will restrict the python version and may developers might not found it familiar

Reply:
@madebr ping